code/ShareCare.php

```diff
@@ -49,10 +49,10 @@ public function updateCMSFields(FieldList $fields)
     {
         $msg = Config::inst()->get('ShareCare', 'cms_message');
         if ($msg) {
-            $fields->addFieldToTab('Root.Share', new LiteralField('ShareCareMessage',
+            $fields->addFieldToTab(_t('ShareCare.TabName'), new LiteralField('ShareCareMessage',
```

Comment:
Please add a default EN string to each `_t()` call, e.g. `_t('ShareCare.TabName', 'Share')` here.